airbyte-integrations/connectors/source-square/integration_tests/configured_catalog.json

```diff
@@ -112,11 +112,11 @@
       "stream": {
         "name": "customers",
         "json_schema": {},
-        "supported_sync_modes": ["full_refresh"],
+        "supported_sync_modes": ["incremental","full_refresh"],
         "source_defined_cursor": true,
         "default_cursor_field": ["id"]
       },
-      "sync_mode": "full_refresh",
+      "sync_mode": "incremental",
       "cursor_field": ["id"],
```

Comment:
@brijeshphilz seems like customer supports `created_at` and `updated_at` https://developer.squareup.com/reference/square/customers-api/list-customers